src/utils/WithStyles/WithStyles.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable react/no-multi-comp */
+import hoistNonReactStatic from 'hoist-non-react-statics';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export default function withStyles(WrappedComponent, defaults) {
+
+    const componentName = WrappedComponent.displayName || WrappedComponent.name;
+
+    class WithStyles extends React.Component {
+        render() {
+            const {
+                customStyles,
+                disableStyles,
+                forwardedRef,
+                ...otherProps
+            } = this.props;
+
+            if (!disableStyles) {
+                if (defaults && defaults.fonts) {
+                    require('fundamental-styles/dist/fonts.css');
+                }
+
+                if (defaults && defaults.icons) {
+                    require('fundamental-styles/dist/icon.css');
+                }
+
+                if (!customStyles && defaults && defaults.cssFile) {
+                    Array.isArray(defaults.cssFile) ?
+                        defaults.cssFile.forEach(file => {
+                            require('fundamental-styles/dist/' + file + '.css');
+                        }) :
+                        require('fundamental-styles/dist/' + defaults.cssFile + '.css');
+                } else {
+                    customStyles;
+                }
+            }
+
+            return (
+                <WrappedComponent
+                    customStyles={customStyles}
+                    disableStyles={disableStyles}
```

Comment:
Do both `customStyles` and `disableStyles` need to be passed through here?  I guess `disableStyles` is useful so components can pass that along to child components, but `customStyles` is only used right here, right?